docs/logging-usage.rst

```diff
@@ -390,72 +391,121 @@ Delete a sink:
-create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
-Logging client.
+It's possible to tie the Python :mod:`logging` module directly into Google
+Stackdriver Logging. There are different handler options to accomplish this.
+To automatically pick the default for your current environment, use
+:meth:`~google.cloud.logging.client.get_default_handler`.
 
 .. code-block:: python
 
     >>> import logging
     >>> import google.cloud.logging # Don't conflict with standard logging
     >>> from google.cloud.logging.handlers import CloudLoggingHandler
     >>> client = google.cloud.logging.Client()
-    >>> handler = CloudLoggingHandler(client)
+    >>> handler = client.get_default_handler()
     >>> cloud_logger = logging.getLogger('cloudLogger')
-    >>> cloud_logger.setLevel(logging.INFO) # defaults to WARN
+    >>> cloud_logger.setLevel(logging.INFO)
     >>> cloud_logger.addHandler(handler)
     >>> cloud_logger.error('bad news')
 
+It is also possible to attach the handler to the root Python logger, so that
+for example a plain ``logging.warn`` call would be sent to Stackdriver Logging,
+as well as any other loggers created. A helper method
+:meth:`~google.cloud.logging.client.setup_logging` is provided
+to configure this automatically.
+
+.. code-block:: python
+
+    >>> import logging
+    >>> import google.cloud.logging # Don't conflict with standard logging
+    >>> client = google.cloud.logging.Client()
+    >>> client.setup_logging(log_level=logging.INFO)
+
 .. note::
 
-    This handler by default uses an asynchronous transport that sends log entries on a background
-     thread. However, the API call will still be made in the same process. For other transport
-     options, see the transports section.
+    To reduce cost and quota usage, do not enable Stackdriver logging
+    handlers while testing locally.
 
-All logs will go to a single custom log, which defaults to "python". The name of the Python
-logger will be included in the structured log entry under the "python_logger" field. You can
-change it by providing a name to the handler:
+You can also exclude certain loggers:
 
 .. code-block:: python
 
-    >>> handler = CloudLoggingHandler(client, name="mycustomlog")
+   >>> client.setup_logging(log_level=logging.INFO,
+                            excluded_loggers=('werkzeug',))
+
+Cloud Logging Handler
+=====================
 
-It is also possible to attach the handler to the root Python logger, so that for example a plain
-`logging.warn` call would be sent to Cloud Logging, as well as any other loggers created. However,
-you must avoid infinite recursion from the logging calls the client itself makes. A helper
-method :meth:`setup_logging <google.cloud.logging.handlers.setup_logging>` is provided to configure
-this automatically:
+If you prefer not to use :meth:`google.cloud.logging.client
+.get_default_handler`, you can directly create a
```

Comment:
Big time :-1: on the line wrap for the `meth` link. Just give it its own line if you must:

```
If you prefer not to use
:meth:`google.cloud.logging.client.get_default_handler`,
you can directly create a
```